docs/developer-docs/integrations/sns/integrating/ledger-integration.md

```diff
@@ -9,108 +9,5 @@ as well as the account balances at the last block of the chain.
-    initial_balances : vec record { Account; nat64 };
-    archive_options : record {
-        num_blocks_to_archive : nat64;
-        trigger_threshold : nat64;
-        max_message_size_bytes : opt nat64;
-        cycles_for_archive_creation : opt nat64;
-        node_max_memory_size_bytes : opt nat64;
-        controller_id : principal;
-    };
-};
-```
-
-Example with `dfx`:
-
-```shell
-dfx deploy icrc1-ledger --argument "(record {
-    token_symbol = \"ABC\";
-    token_name = \"My example token\";
-    minting_account = record { owner = principal \"rrkah-fqaaa-aaaaa-aaaaq-cai\" };
-    transfer_fee = 10_000;
-    metadata = vec {};
-    initial_balances = vec {};
-    archive_options = record {
-        num_blocks_to_archive = 10_000;
-        trigger_threshold = 20_000;
-        controller_id = principal \"rrkah-fqaaa-aaaaa-aaaaq-cai\";
-        cycles_for_archive_creation = opt 4_000_000_000_000;
-    };
-})"
-```
-
-## Usage
-
-Here are the supported methods from the ledger canister:
-
-```
-icrc1_name : () -> (text) query;
-```
-- This method returns the name of the token (e.g., MyToken).
-
-```
-icrc1_symbol : () -> (text) query;
-```
-- This method returns the symbol of the token (e.g., ICP).
-
-```
-icrc1_decimals : () -> (nat8) query;
-```
-- This method returns the number of decimals the token uses (e.g., 8 means to divide the token amount by 100000000 to get its user representation).
-
-```
-icrc1_metadata : () -> (vec record { text; Value }) query;
-```
-- This method returns the list of metadata entries for this ledger. 
-
-```
-icrc1_total_supply : () -> (Tokens) query;
-```
-- This method returns the total number of tokens on all accounts except for the minting account.
-
-```
-icrc1_fee : () -> (Tokens) query;
-```
-- This method returns the default transfer fee.
-
-```
-icrc1_minting_account : () -> (opt Account) query;
-```
-- This method returns the minting account if this ledger supports minting and burning tokens.
-
-```
-icrc1_balance_of : (Account) -> (Tokens) query;
-```
-- This method returns the balance of the account given as an argument.
-
-```
-icrc1_transfer : (TransferArg) -> (TransferResult);
-```
-- This method transfers some tokens between two accounts. The transfer fees are taken from the caller's account in addition to the transferred amount.
-
-```
-icrc1_supported_standards : () -> (vec record { name : text; url : text }) query;
-```
-- This method returns the list of standards this ledger implements.
-
-## Candid reference file
-
-Please check the [Candid file](https://gitlab.com/dfinity-lab/public/ic/-/blob/master/rs/rosetta-api/icrc1/ledger/icrc1.did) of the ledger canister for the interface details.
-
-
+For instructions on how to install the ICRC-1 Ledger follow [ICRC-1 Ledger Deployment](/developer-docs/integrations/icrc-1/deploy-new-token.md)
+For instructions on how to interact with the ICRC1-Ledger follow [ICRC-1 Ledger Interaction](/developer-docs/integrations/icrc-1/index.md)
```

Comment:
```suggestion
For instructions on how to interact with the ICRC-1 Ledger follow [ICRC-1 Ledger Interaction](/developer-docs/integrations/icrc-1/index.md)
```

(I assume his is correct based on how it is used a line above)

Reply:
Yes, I checked that `developer-docs/integrations/icrc-1/deploy-new-token.md` works. It's easier to have it in one place.